front/components/data_source/DeleteStaticDataSourceDialog.tsx

```diff
@@ -34,8 +34,8 @@ export function DeleteStaticDataSourceDialog({
   const message =
     dataSourceUsage === undefined
       ? `Are you sure you want to permanently delete ${name}?`
-      : dataSourceUsage > 0
-        ? `${dataSourceUsage} assistants currently use ${name}. Are you sure you want to remove?`
+      : dataSourceUsage.count > 0
+        ? `${dataSourceUsage} assistants currently use ${name}: ${dataSourceUsage.agentNames.join(", ")}. Are you sure you want to remove?`
```

Comment:
Let's use elipsis here? We have `truncate` function I think.

Reply:
I rather show them all to be honest, but does the dialog handle it ?

Reply:
That's my fear.. I'm afraid the layout is gonna break..

Reply:
I'll test, thanks for raising the point.

Reply:
We're good :)
<img width="489" alt="image" src="https://github.com/user-attachments/assets/ad99285d-e2e2-43d7-b51e-c93de099dcef">